src/ssl/PeekingPeerConnector.cc

```diff
@@ -69,9 +69,9 @@ Ssl::PeekingPeerConnector::checkForPeekAndSplice()
 {
     handleServerCertificate();
 
-    ACLFilledChecklist *acl_checklist = new ACLFilledChecklist(
+    const auto acl_checklist = new ACLFilledChecklist(
         ::Config.accessList.ssl_bump,
-        request.getRaw(), nullptr);
+        request.getRaw());
```

Comment:
Please take advantage of the changes here to make this a one-liner:
```suggestion
    const auto acl_checklist = new ACLFilledChecklist(::Config.accessList.ssl_bump, request.getRaw());
```

Reply:
Ok.